spec/controllers/ideas_controller_spec.rb

```diff
@@ -0,0 +1,42 @@
+require 'rails_helper'
+
+RSpec.describe IdeasController, :type => :controller do
+  describe '#index' do
+    it 'assigns ideas as @ideas' do
+      get :index
+      expect(assigns(:ideas)).to eq(Idea.all)
+    end
+
+    it {
+      get :index
+      should render_template('index')
+    }
+  end
+
+  describe '#new' do
+    it 'assigns a blank unsaved idea as @idea' do
+      get :new
+      expect(assigns(:idea)).to be_a_new Idea
+    end
+
+    it {
+      get :new
+      should render_template('new')
+    }
+  end
+
+  describe '#create' do
+    # it 'creates a new idea' do
+    #   expect(
+    #     post :create, { idea: { title: 'hello hello', description: 'hello hello again' } }
+    #   ).to change(Idea.count).by 1
+    # end
+
+    # it {
+    #   get :create
+    #   # redirect_to
+    # }
```

Comment:
Commenting out code in longer running projects is generally frowned upon. We have git and a compete history of our project so we can always look through that for the code we removed if we decide we needed it. Also, don't you want to test this? ;)